app/javascript/dashboard/routes/dashboard/conversation/labels/LabelBox.vue

```diff
@@ -100,13 +113,24 @@ export default {
         // Ignore error
       }
     },
-    onEdit() {
-      this.isEditing = true;
+
+    toggleLabels() {
+      this.showSearchDropdownLabel = !this.showSearchDropdownLabel;
     },
-    closeEditModal() {
-      bus.$emit('fetch_conversation_stats');
-      this.isEditing = false;
+
+    addItem(value) {
+      this.selectedLabels.push(value.title);
+      this.onUpdateLabels(this.selectedLabels);
+    },
+
+    removeItem(value) {
+      this.selectedLabels.filter(label => label.title === value.title);
```

Comment:
`this.selectedLabels.filter(label => label.title === value.title);` will return a new array without `value.title`. Assign the result this to a new `const` and assign it to  `this.selectedLabels`. Then call `this.onUpdateLabels(this.selectedLabels);`